lib/inets/src/inets_app/inets.erl

```diff
@@ -47,10 +47,12 @@
 %% Description: Starts the inets application. Default type
 %% is temporary. see application(3)
 %%--------------------------------------------------------------------
-start() -> 
+start() ->
+    application:ensure_all_started(ssl),
     application:start(inets).
```

Comment:
Actually I think it would make more sense to do application:ensure_all_started([inets, ssl]).